docs/quickstart/Yarn.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: page
+title: "Install"
+description: "This page will help you get started and will guide you through installing Apache Zeppelin and running it in the command line."
+group: quickstart
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# Zeppelin on Yarn
+
+Zeppelin can run on [Yarn](https://hadoop.apache.org/submarine). Zeppelin server runs in local host, it creates docker container for individual interpreter. All interpreters run in yarn (The local mode of the spark interpreter also runs on the yarn).
+
+Key benefits are
+
+ - Interpreter scale-out
+ - Docker container environmental isolation
+ - Users can easily install python and R libraries
+
+## Prerequisites
+
+ - YARN 3.2.0+, You can use [Submarine-installer](https://github.com/hadoopsubmarine/submarine-installer), Deploy Docker and runtime environments.
```

Comment:
Why it would depend on submarine ?

Reply:
Do not depend on submarine, Need depend on hadoop-3.2+.
But the hadoop-3.2+ version is not yet integrated into CDH, ambari.
So the hadoop submarine team provides the installer for the hadoop-3.2+ runtime environment.

Reply:
Ok, Some word descriptions in the document, Need to update.